frontend/src/Editor/LeftSidebar/SidebarComment.jsx

```diff
@@ -7,15 +7,17 @@ export const LeftSidebarComment = ({ toggleComments, appVersionsId }) => {
   return (
     <LeftSidebarItem
       commentBadge
-      tip="toggle comments"
+      tip={appVersionsId ? 'toggle comments' : 'At least one version of the app should be deployed'}
```

Comment:
Can we change the tooltip to:
```suggestion
      tip={appVersionsId ? 'toggle comments' : 'Comments section will be available once you save this application'}
```

Reply:
@sherfin94 yeah sure